app/assets/stylesheets/user-profile-header.scss

```diff
@@ -229,12 +229,14 @@
       @media screen and (min-width: 430px) {
         font-size: 1.15em;
         padding: 15px 0px;
+        padding-right: 15px;
```

Comment:
as you already have a padding definition, maybe you can just keep that. Right now you have padding top and bottom to be 15 and right and left to be 0. you can use `padding: 15px 15px 15px 0` to use one rule

Reply:
@rhymes finished update.